README.md

```diff
@@ -43,37 +42,38 @@ The following tools should be installed in your local machine to install the RAD
 
 ### Prepare
 1. Clone the repository to your local machine by using following command.
-It will download the dependent helm charts from Confluent platform for components such as Kafka and Zookeeper.
+    It will download the dependent helm charts from Confluent platform for components such as Kafka and Zookeeper.
+
```

Comment:
Is the following line still correct? Do we still need `--recurse-submodules` here?